pom.xml

```diff
@@ -59,7 +59,19 @@
       <artifactId>pac4j-core</artifactId>
       <version>${pac4j.version}</version>
     </dependency>
+    <!-- provided -->
+    <dependency>
+      <groupId>org.pac4j</groupId>
+      <artifactId>pac4j-jwt</artifactId>
+      <version>${pac4j.version}</version>
+      <scope>provided</scope>
+    </dependency>
     <!-- tests -->
+    <dependency>
```

Comment:
Is this really for tests? In that case, we should have the `<scope>test</scope>`, right?

Reply:
Oh, I missed the scope 😞 
Yes, it dependency needed only for tests (`JWTTestDataGenerator`).
Thank you, Jerome! 👍

Reply:
The build fails. It looks like you need the `nimbus-jose-jwt` dependency for compilation as well and you should remove the `test` scope...

Reply:
`nimbus-jose-jwt` (5.4) is transitive dependency of `pac4j-jwt`.
But version 5.4 hasn't `*KeyGenerator` classes that using in `JWTTestDataGenerator`.
I set `provided` scope for this dependency. It will be correct for compile and test classpaths.

Reply:
OK. Good. Merging...